examples/universal-cnf/vppagent/pkg/ucnf/ucnf_nse.go

```diff
@@ -18,12 +97,13 @@ func (ucnf *UcnfNse) Cleanup() {
 	ucnf.processEndpoints.Cleanup()
 }
 
-func NewUcnfNse(configPath string, verify bool, backend config.UniversalCNFBackend, ceAddons config.CompositeEndpointAddons) *UcnfNse {
+func NewUcnfNse(configPath string, verify bool, backend config.UniversalCNFBackend, ceAddons config.CompositeEndpointAddons, ctx *context.Context) *UcnfNse {
 	cnfConfig := &nseconfig.Config{}
 	f, err := os.Open(configPath)
 	if err != nil {
 		logrus.Fatal(err)
 	}
+	defer func() { _ = f.Close() }()
```

Comment:
You could possibly at least log the error, instead of simply ignoring it.

Reply:
done